frontend/packages/shared/src/components/LeftNavigationBar/GoBackButton/GoBackButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactNode } from 'react';
+import classes from './GoBackButton.module.css';
+import cn from 'classnames';
+import { ArrowLeftIcon } from '@navikt/aksel-icons';
+import { Paragraph } from '@digdir/design-system-react';
+
+export type GoBackButtonProps = {
+  /**
+   * Classname for navigation element
+   */
+  navElementClassName: string;
+  /**
+   * Function to be executed when clicking the back button
+   * @returns void
+   */
+  onClickBackButton: () => void;
+  /**
+   * Text on the back button
+   */
+  backButtonText: string;
```

Comment:
Komponenten i seg selv er en "GoBackButton" og tenker det ikke er nødvendig å gjenta navnet i props. :)
```suggestion
  onClick: () => void;
  /**
   * Text on the back button
   */
 text: string;
```